scoping_document.md

```diff
@@ -167,14 +167,14 @@ The corresponding acceptance criteria supplement the specification of the requir
 ### Technical Support
 | # of user story ID | User story | Acceptance criteria |
 |-----------------|------------|--------------------|
-| E08.01 | As an app user, I want to be able to contact a hotline, so that I can resolve any technical problems with the app. | 1. The phone number of the technical hotline is stored in the app. |
+| E08.01 | As an app user, I want to be able to contact a hotline, so that I can resolve any technical problems with the app. | 1. The phone number of the technical hotline is shown in the app. |
 
 ### Accessibility
 | # of user story ID | User story | Acceptance criteria |
 |-----------------|------------|--------------------|
-| E09.01 | As an app user, I want it to have audio output, so that I can use the app even if I have impaired vision (or am blind), for example. | 1. Accessibility regarding audio output is provided depending on the options available in the respective operating system. |
-| E09.02 | As an app user, I want to have good contrast settings, modifiable font sizes, and an easily readable font, so that I can read the texts in the app easily. | 1. Accessibility regarding contrast and font size/type is provided depending on the options available in the respective operating system. |
-| E09.03 | As an app user, I want to have the content in simplified language, so that I can easily understand how to use the app and why I should use it. | 1. The texts and languages are defined by the ordering party. |
+| E09.01 | As an app user, I want the app to provide audio output, so that I can use the app even if, for example, I have impaired vision or I am blind. | 1. Accessibility regarding audio output is provided depending on the options available in the respective operating system. |
```

Comment:
> so that I can use the app even if, for example, I have impaired vision or I am blind.

could be

> so that I can use the app even if, for example, I have impaired vision or blindness

Reply:
Fixed.